packages/next/build/webpack/config/blocks/css/index.ts

```diff
@@ -173,46 +218,62 @@ export const css = curry(async function css(
-                  mode: 'pure',
-                  // Generate a friendly production-ready name so it's
-                  // reasonably understandable. The same name is used for
-                  // development.
-                  // TODO: Consider making production reduce this to a single
-                  // character?
-                  getLocalIdent: getCssModuleLocalIdent,
-                },
+              importLoaders: 1,
+              sourceMap: true,
+              onlyLocals: ctx.isServer,
+              modules: {
+                // Disallow global style exports so we can code-split CSS and
+                // not worry about loading order.
+                mode: 'pure',
+                // Generate a friendly production-ready name so it's
+                // reasonably understandable. The same name is used for
+                // development.
+                // TODO: Consider making production reduce this to a single
+                // character?
+                getLocalIdent: getCssModuleLocalIdent,
               },
             },
+            'sass-loader'
+          ),
+        },
+        // Opt-in support for SASS (using .scss or .sass extensions).
+        // By default we support SASS Modules with the
+        // extensions .module.scss or .module.sass
+        {
+          // CSS Modules should never have side effects. This setting will
+          // allow unused CSS to be removed from the production build.
+          // We ensure this by disallowing `:global()` CSS at the top-level
+          // via the `pure` mode in `css-loader`.
+          sideEffects: false,
+          test: /\.module\.(scss|sass)$/,
+          // CSS Modules are only supported in the user's application. We're
+          // not yet allowing CSS imports _within_ `node_modules`.
+          issuer: {
+            include: [ctx.rootDirectory],
+            exclude: /node_modules/,
+          },
 
-            // Compile CSS
+          use: getModuleStyleLoader(
             {
-              loader: require.resolve('postcss-loader'),
-              options: {
-                ident: '__nextjs_postcss',
-                plugins: postCssPlugins,
-                sourceMap: true,
+              importLoaders: 2,
+              sourceMap: true,
+              onlyLocals: ctx.isServer,
+              modules: {
+                // Disallow global style exports so we can code-split CSS and
+                // not worry about loading order.
+                mode: 'pure',
+                // Generate a friendly production-ready name so it's
+                // reasonably understandable. The same name is used for
+                // development.
+                // TODO: Consider making production reduce this to a single
+                // character?
+                getLocalIdent: getCssModuleLocalIdent,
               },
             },
-          ] as webpack.RuleSetUseItem[]).filter(Boolean),
+            'sass-loader'
```

Comment:
Must be resolved.

Reply:
Is resolved in the loader compiling function https://github.com/zeit/next.js/pull/10133/files#diff-0df73c8e15d5a4ab2409907c614af8ebR170